<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlimeManager</class>
 <widget class="QDialog" name="SlimeManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>629</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SlimeManager</string>
  </property>
  <widget class="QTabWidget" name="SlimeManager_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="SlimeManager">
    <attribute name="title">
     <string>SlimeManager</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1311</width>
       <height>591</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mixxx/Final Folder</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editFinalDirectory">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>100</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listFinalDirectory">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>150</y>
        <width>261</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>330</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="listUrlDownloaded">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>150</y>
        <width>261</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefreshFolders">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>60</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh Folders</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>130</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>In Mixxx Folder</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>330</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>130</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Downloaded / Complete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveToFinalDirectory">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>120</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Move to Mixxx</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>631</width>
        <height>571</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <widget class="QListWidget" name="listSearchResults">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>601</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Search Results</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSearchYoutube">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="editSearchYoutube">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>601</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Search YouTube</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Downloader</string>
       </attribute>
       <widget class="QListWidget" name="listUrlInQueue">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>601</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Downloading...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>60</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Add URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="editAddUrl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>601</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAddUrlToQueue">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>191</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Smart Downloader</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAddUrlToQueue_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>60</y>
          <width>191</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Youtube Downloader</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAddUrlToQueue_3">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>60</y>
          <width>191</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>General Downloader</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Converter">
    <attribute name="title">
     <string>File Converter</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Root Directory</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listConvertExploreFiles">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>381</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File Directory</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editConvertRootDirectory">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConvertListBack">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>110</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupConverter">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>140</y>
       <width>361</width>
       <height>421</height>
      </rect>
     </property>
     <property name="title">
      <string>Converter</string>
     </property>
     <widget class="QCheckBox" name="toggleConvertMP3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MP3 (Audio)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="toggleConvertMP4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MP4 (Video)</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblInputPath">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>/path/to/file/</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>File To Convert:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConvert">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>380</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Destination:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblOutputPath">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>/path/to/output/</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="QRCode">
    <attribute name="title">
     <string>QR Code</string>
    </attribute>
    <widget class="QPushButton" name="btnQrCreate">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>330</y>
       <width>421</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Make QR Code</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="imageViewQrLogo">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>110</y>
       <width>301</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="editTextQrData">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>401</width>
       <height>78</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>20</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Logo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>200</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="editTextQrColor">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="editTextQrLogoPath">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>40</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQrAddLogo">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>70</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Logo</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="editTextQrExportPath">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>290</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>260</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Export Location</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="editTextQrName">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
